internal/run.go

```diff
@@ -132,11 +136,22 @@ func (proc *cmdProcess) Start() error {
 	return proc.cmd.Start()
 }
 
-func (proc *cmdProcess) Kill() error {
+func (proc *cmdProcess) Stop() error {
```

Comment:
b/c of the timeout, this really StopAndThenKill(), instead of either Stop or Kill. Maybe it's Kill(gracePeriod)?